CordovaLib/Classes/Public/CDVAppDelegate.m

```diff
@@ -79,8 +79,16 @@ - (BOOL)application:(UIApplication*)application openURL:(NSURL*)url sourceApplic
         return NO;
     }
 
+    NSDictionary *openURLData = @{
+            @"url": url,
+            @"sourceApplication": sourceApplication,
+            @"annotation": annotation
```

Comment:
`annotation` could be `nil` here leading to unhandled exception resulting in a crash